src/lib.rs

```diff
@@ -147,8 +152,8 @@ mod tests {
 
     #[bench]
     fn bench_remem(b: &mut Bencher) {
+        let pool = Pool::<Vec<usize>>::new(1024, || Vec::new(), |v| v.clear());
```

Comment:
Moved this out so that the pool is persisted between runs. Also removes the initialization from being factored into the test run. Tested this with the "current" bench, but didn't make any difference really (: